ci-operator/step-registry/firewatch/ipi/aws/firewatch-ipi-aws-workflow.yaml

```diff
@@ -2,7 +2,8 @@ workflow:
   as: firewatch-ipi-aws
   steps:
     pre:
-    - chain: ipi-aws-pre
+      - ref: install-operators-verify
```

Comment:
My only issue with this is there are some scenarios that use the `firewatch-ipi-aws` workflow that do not use `install-operators` so the `OPERATORS` variable will not be present in the config. For example, [serverless](https://github.com/openshift/release/blob/master/ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-main__ocp4.16-lp-interop.yaml#L217) does not use the ref (plus there are a couple of scenarios that use this that are not operators).

We may just need to add this to the scenarios where it applies? The grate thing about this being a ref is that we will be able to add a firewatch rule specifically for failures where the operator is not in the catalog for a pre-release OCP build. 😄

Reply:
Can we define which scenarios will default to using the `firewatch-ipi-aws` workflow and apply this ref to them? Additionally, can we define which scenarios are exceptions (do not include `install-operators` ref; do not use operators, or use operators that may be in a different catalog)?

Reply:
Providing info for future interest:

Within our internal discussions, we verified that using the `OPERATORS` env var is 100% optional, therefore scenarios that uses the `firewatch-ipi-aws` workflow and apply this ref doesn't get effected

See POC:
Job [periodic-ci-quarkus-qe-quarkus-openshift-interop-main-quarkus-ocp4.16-lp-interop-quarkus-interop-aws](https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/50255/rehearse-50255-periodic-ci-quarkus-qe-quarkus-openshift-interop-main-quarkus-ocp4.16-lp-interop-quarkus-interop-aws/1775150998822588416#1:build-log.txt%3A34)
- Uses the workflow
- Doesn't use `install-operators` ref for installing the Operator
- Passed this step with:
`Step quarkus-interop-aws-install-operators-verify succeeded after 8s.`
(Job aborted manually before executing cluster provision)